conftest.py

```diff
@@ -16,6 +16,8 @@
 from concrete.ml.common.utils import (
     SUPPORTED_FLOAT_TYPES,
     all_values_are_floats,
+    check_allclose_and_shape,
```

Comment:
"check" semantic is more for fixture or stuff that raises errors, I would rename these two ones by something like `array_isclose_and_same_shape`  (same for allclose)